src/main/scala/uk/gov/nationalarchives/tdr/api/http/GraphQLServer.scala

```diff
@@ -25,21 +27,25 @@ import scala.util.{Failure, Success}
 
 object GraphQLServer {
 
+  private val logger = Logger(classOf[GraphQLServer])
+
+  private def handleException(marshaller: ResultMarshaller, errorCode: String, message: String): HandledException = {
+    val node = marshaller.scalarNode(errorCode, "String", Set.empty)
+    val additionalFields = Map("code" -> node)
+    logger.warn(s"$message, Codes: ${additionalFields}")
```

Comment:
Minor thing, but I'd just include the `errorCode` instead of `additionalFields`. The latter is just the structure we need to put it in for Sangria to add it to the GraphQL response - the error code is the important thing to have in the logs.